CMakeLists.txt

```diff
@@ -42,15 +42,15 @@ endif ()
 set(MMSEQS_CXX_FLAGS "-std=c++0x")
 if (CMAKE_SYSTEM_PROCESSOR MATCHES "^(arm.*|ARM.*)")
     set(ARM 1)
-    set(MMSEQS_CXX_FLAGS "${MMSEQS_CXX_FLAGS} -DNEON=1 -DSSE=1 -fsigned-char -mfpu=neon")
+    set(MMSEQS_CXX_FLAGS "${MMSEQS_CXX_FLAGS} -fsigned-char -mfpu=neon")
```

Comment:
I think `-mfpu=neon` should be set in CI config, or elsewhere, but not for all ARM* users